pkg/processor/sourceprocessor/README.md

```diff
@@ -67,3 +67,20 @@ processors:
       namespace: "kube-system"
       pod: "custom-pod-.*"
 ```
+
+## Pod annotations
+
+The following [Kubernetes annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+can be used on pods:
+
+- `sumologic.com/exclude` - records from this pod are dropped
+- `sumologic.com/include` - overrides `exclude` processor settings, records from this pod are not checked against those regexes
```

Comment:
* What are the possible values for these labels? When the filtering occurs? When this is strictly `true` or ... ? 
* `sumologic.com/include` overrides `sumologic.com/exclude` ? or both `exclude` and `sumologic.com/exclude`?

Reply:
This pull request is not about the `sumologic.com/exclude` or `sumologic.com/include` pod annotations. I only included these docs because I was copying the whole section abot pod annotations from [Fluentd docs](https://github.com/SumoLogic/sumologic-kubernetes-fluentd/tree/main/fluent-plugin-kubernetes-sumologic#pod-annotations) - it's probably better than nothing, but still far from perfect.

The docs on this processor are still in many ways not sufficient and this pull request is not aiming to make them perfect.

I fully agree with the remarks above, but I believe they should be resolved in a separate pull request.

Reply:
https://github.com/SumoLogic/sumologic-otel-collector/issues/259 :) alrighty then

Reply:
plesase create issue then

Reply:
:) https://github.com/SumoLogic/sumologic-otel-collector/pull/257#discussion_r716615282

Reply:
:rock: